autorest/codegen/__init__.py

```diff
@@ -124,6 +124,8 @@ def _build_package_dependency() -> Dict[str, str]:
             "dependency_azure_mgmt_core": "azure-mgmt-core<2.0.0,>=1.3.0",
             "dependency_azure_core": "azure-core<2.0.0,>=1.24.0",
             "dependency_msrest": "msrest>=0.7.1",
+            "dependency_isodate": "isodate<1.0.0,>=0.6.1",
```

Comment:
hey @changlong-liu sorry if I wasn't clear, we should keep the same code, just `bandit: ignore` those two lines. I'll go ahead and do another PR. Sorry about that!

Reply:
I see, agree that changing behavior is somehow dangerous